NCP/openncp-configuration/tm.properties

```diff
@@ -109,22 +104,13 @@ tm.documenttype.mro=56445-0
 ## epSos code for HCER CDA document
 tm.documenttype.hcer=34133-9
 
-################################################################## Audit Trail Configuration #
-
-## Audit Trail enabled/disabled
-tm.audittrail.enabled=false
-
-## Audit Trail Event Log - The number of transaction including the epsos- prefix
-tm.audittrail.transactionnumber=epsos-94
-
-## Audit Trail Event Log - The IP Address of the target Gateway
-tm.audittrail.targetip=
+###### Audit Trail Configuration
 
 ## Audit Trail Audit Service - The facility number according to log4j
 tm.audittrail.facility=13
 
-## Audit Trail Audit Service - The severity of the message 
+## Audit Trail Audit Service - The severity of the message
 tm.audittrail.severity=2
 
 #NCP-side that is calling the transformation service
-tm.ncp.side=NCP_A
+tm.ncp.side=NCP_B
```

Comment:
Eeh, this might be interesting... We will definitely be calling it from `NCP_A`, but it would be nice if we could call it from both sides...